### PR TITLE
Amp-playbuzz item-id support 

### PR DESCRIPTION
**Pull request by buzzdan**
- Added Amp-playbuzz item-id support.
- Item Url is still supported

This is an important feature that will help publishers to auto-convert their embeded playbuzz  items on their site to AMP

### CODE REVIEW

File: extensions/amp-playbuzz/0.1/utils.js [hunk 1]
```diff
@@ -137,3 +141,18 @@ export function composeEmbedUrl(options) {
   });
   return embedUrl;
 }
+
+export function composeItemSrcUrl(src, itemId) {
+  const DEFAULT_BASE_URL = '//www.playbuzz.com/';
+  const sanitizeUrl = localtion =>
```
Comment (jridgewell) [inline extensions/amp-playbuzz/0.1/utils.js]: This can be hoisted out. Also, spelling.

File: extensions/amp-playbuzz/0.1/utils.js [hunk 2]
```diff
@@ -137,3 +141,18 @@ export function composeEmbedUrl(options) {
...
+    removeFragment(localtion.href)
+      .replace(localtion.protocol, ''); //remove scheme (cors) & fragment
+
+  const iframeSrcUrl = itemId ?
+    DEFAULT_BASE_URL + 'item/' + itemId :
+    [src]
```
Comment (jridgewell) [inline extensions/amp-playbuzz/0.1/utils.js]: Why the array and map? `sanitizeUrl(parseUrl(src))`.

File: extensions/amp-playbuzz/0.1/validator-amp-playbuzz.protoascii [hunk 3]
```diff
@@ -57,8 +57,12 @@ tags: {  # <amp-playbuzz>
   tag_name: "AMP-PLAYBUZZ"
   also_requires_tag: "amp-playbuzz extension .js script"
   attrs: {
-    name: "data-comments"
```
Comment (jridgewell) [inline extensions/amp-playbuzz/0.1/validator-amp-playbuzz.protoascii]: /cc @Gregable

**Comment by dvoytenko** [comment]
@buzzdan Thanks! All good on my side, just a couple of comments from @jridgewell and request to confirm validator changes to @Gregable

**Comment by Gregable** [comment]
@dvoytenko looks good to me

**Comment by buzzdan** [comment]
@jridgewell @dvoytenko thanks for the review, i've pushed the requested change ;)

**Comment by dvoytenko** [comment]
Thanks all! Merged.

**Pull request merged**